bosh_cli/lib/cli/commands/events.rb

```diff
@@ -41,6 +41,9 @@ def show_events
           row << event.fetch('task', '-')
           row << event.fetch('deployment', '-')
           row << event.fetch('instance', '-')
+          if !event.key?('context') || event['context'] == 'null' || event['context'] == nil
```

Comment:
With the 4f5ef6a change, this could be simplified since the value will no longer be `null`.